squash_command_test.go

```diff
@@ -125,19 +127,31 @@ var ItSquashesPR = func(context WebhookTestContext, pr *github.PullRequest) {
 		})
 	})
 
-	Context("with autosquash succeeding", func() {
+	Context("with autosquash and push failing for a different reason", func() {
```

Comment:
It's actually for cases where the thing fails _for a reason other than a squash conflict_. So https://github.com/salemove/github-review-helper/pull/24/commits/9d3e088ecad3e4d5a9fb55b19e1327f27f60a87b?